platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -429,6 +429,53 @@ public void run() {
     }
   }
 
+  /**
+   * Same as the method above but skipping database file packing for performance reasons.
```

Comment:
Could you link the method instead with `{@link com.mapbox.mapboxsdk.offline.OfflineRegion#delete(OfflineRegion.OfflineRegionDeleteCallback)}`?